packages/manager/package.json

```
@@ -86,7 +86,7 @@
     "zxcvbn": "^4.4.2"
   },
   "scripts": {
-    "start": "concurrently --raw \"vite\" \"tsc --watch --preserveWatchOutput\"",
+    "start": "concurrently --raw \"NODE_OPTIONS='--max-old-space-size=4096' vite\" \"NODE_OPTIONS='--max-old-space-size=4096' tsc --watch --preserveWatchOutput\"",
```

From what I observed, it is just the `tsc` process that has the OOM issue. I think we'd likely fix the issue by just apply the fix to the `tsc` command, but at the same time, I don't think it will hurt having it on the Vite processes too

```suggestion
    "start": "concurrently --raw \"vite\" \"NODE_OPTIONS='--max-old-space-size=4096' tsc --watch --preserveWatchOutput\"",
```

---

Might be nice to update the "start" command to have `-n vite,tsc` so we can better understand which process is crashing but we don't have to

For example:
```
"start": "concurrently -n vite,tsc \"vite\" \"tsc --watch --preserveWatchOutput\"",
```

---

I think this could be done locally to debug the underlying issues in a separate task - we should have a separate ticket to investigate this - this PR facilitates development for now